README.md

```diff
@@ -1,12 +1,16 @@
-[Codecov][1] Ruby Example
-=========================
+# [Codecov][1] Ruby Example
+## Guide
+### Travis Setup
```

Comment:
@TomPed To make this a bit more agnostic, what if we just made this an `##` and title it "CI setup" and add a  descriptive line of text mentioning that this is for travis CI but that most modern CIs with a `.yml` file should work appropriately. potentially linking to additional popular CI services like CircleCI and Jenkins.

additionally, i would removed the line "guide" above it, i think? It should be implicitly understood this is a guide from reading it, in my opinion